src/blocks/EventAccordion/Event.js

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import styled from "styled-components";
+import { string, func, bool, shape } from "prop-types";
+import { Button} from "../../components/Button"
+import { EventType } from "../../components/types";
+import { mediumAndUp } from "../../theme/mediaQueries";
+
+import ActionArea from "./ActionArea";
+import Badge from "./Badge"
+import BottomSheet from "./BottomSheet";
+import Chevron from "./Chevron";
+import Date from "./Date";
+import Ellipsis from "./Ellipsis";
+import Layout from "./Layout"
+import RevealAnimation from "./RevealAnimation";
+import Tile from "../Tile";
+
+import formatOnSaleText from "./utils/text";
+import shouldAnimate from "./utils/animation";
+
+const Wrapper = styled(Tile)`
+  display: flex;
+  flex-direction: row;
+  padding: 4px 0px;
+
+  .chevron { display: none}
+  .active-event-text { display: none;}
+  ${mediumAndUp` .chevron{ display: inline; }`}
+
+`;
+
+const Hoverable = styled(RevealAnimation)`
+  flex: 1 1;
+  display: flex;
+  padding-bottom: 14px;
+  padding-top: 14px;
+  padding-left: 8px;
+  cursor: pointer;
+
+  &&:hover {
+    background: lavender;
+  }
+
+`;
+
+const Event = ({
```

Comment:
I'm thinking maybe we should move all of this `EventAccordion` logic into `aurora-extensions` 🤔

Reply:
Yes, that's a good idea.